diff --git a/src/Plugin/Block/SpcSdgSummary.php b/src/Plugin/Block/SpcSdgSummary.php
index 9896808..84a8485 100755
--- a/src/Plugin/Block/SpcSdgSummary.php
+++ b/src/Plugin/Block/SpcSdgSummary.php
@@ -19,7 +19,7 @@ use Drupal\Core\Url;
  * @Block(
  *   id = "spc_sdg_summary",
  *   admin_label = @Translation("SDG Chart Summary"),
- *   context = {
+ *   context_definitions = {
  *     "node" = @ContextDefinition("entity:node", label = @Translation("Node"))
  *   }
  * )
@@ -320,9 +320,9 @@ EOS;
 
     foreach ($entity_storage->loadMultiple($entity_ids) as $node) {
 
-      if ($node->hasTranslation($langcode)) {
-        $node = $node->getTranslation($langcode);
-      }
+      //if ($node->hasTranslation($langcode)) {
+      //  $node = $node->getTranslation($langcode);
+      //}
 
       $goals[] = $node;
     }
@@ -445,3 +445,4 @@ EOS;
   }
 
 }
+
